cli/tests/e2e/test_check.py

```diff
@@ -599,7 +599,13 @@ def test_max_memory(run_semgrep_in_tmp: RunSemgrep, snapshot):
     snapshot.assert_match(stderr, "error.txt")
 
 
+# It seems that OCaml 5.0 does not use the system stack https://github.com/ocaml-multicore/docs/blob/main/ocaml_5_design.md#stack-layout
+# note that it says it manages its own runtime stack, which are managed in fibers
+# fiber stacks are allocated w/ mallloc + free aka its in the heap.
+# So I thionk this test is no longer valid as of OCaml 5.0. The exception is C calls
+# but ocaml handles the runtime stack switching to system stack for them so we should be good
 @pytest.mark.slow
+@pytest.mark.skip(reason="OCaml 5.0 does not use system stack")
```

Comment:
@brendongo @aryx This makes sense right?

Reply:
also possibly @mjambon since he's usually up to date on his ocaml lore

Reply:
@brandonspark (not brendongo)

Reply:
Adding also @ihji who investigated a bit also switching to ocaml 5.0